File: openapi/code_samples/csharp/api_company_filter.cs
using Microsoft.AspNetCore.Mvc;
using Swashbuckle.AspNetCore.Annotations;

[ApiController]
[Route("api/[controller]")]
public class CompanyController : ControllerBase
{
    [HttpGet]
    [Route("list-indicators")]
    [SwaggerOperation(Summary = "Filters companies based on various indicators.")]
    [ProducesResponseType(typeof(SearchCompanyResponse[]), 200)]
    [ProducesResponseType(typeof(Fault), 400)]
    public IActionResult ListIndicators([FromQuery] CompanyFilterCriteria companyFilterCriteria)
    {
        // Your implementation here
        return Ok();
    }
}

File: openapi/code_samples/python/api_company_filter.py
from flask import Flask, request, jsonify

app = Flask(__name__)

@app.route('/api/company/list-indicators', methods=['GET'])
def list_indicators():
    company_filter_criteria = request.args.get('companyFilterCriteria')
    # Your implementation here
    return jsonify({}), 200

if __name__ == '__main__':
    app.run()
